### ISSUE TITLE
Inability to showcase an ImageView within a GridView

### ISSUE THREAD

**Issue by naphtul** [amlcurran/ShowcaseView#39]
Scenario:
1. Have a GridView with multiple ImageView.
2. Try to showcase the ImageView at position 0.

Expected result:
ShowcaseView is able to showcase the first ImageView in the GridView by recognizing the object and its location of the screen.

Actual result:
I am unable to get ShowcaseView to recognize the ImageView within the GridView.

Additional Info:
I am loading multiple instances of the same fragment using ActionBarSherlock (i.e. SherlockFragment).
I am able to work around the problem by X and Y, but it is not optimal, as different layouts yield different results.
![screenshot_2013-05-04-22-07-42](https://f.cloud.github.com/assets/4341220/462916/917ce85e-b4f3-11e2-8260-3d4176aedfc2.png)
Here's a short version of my code:
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
            Bundle savedInstanceState) {

```
    v = inflater.inflate(R.layout.activity_grid_fragment, container, false);

    gv = (GridView) v.findViewById(R.id.gridView);
    gv.setAdapter(new ImageAdapter(v.getContext(), args));
    registerForContextMenu(gv);
    gv.setOnItemClickListener(new OnItemClickListener() {
        public void onItemClick(AdapterView<?> parentView, View iv, int position, long id) {
            .
            .
        }
    });

    if (firstLaunch ){
        showShowcase();
    }
    return v;
}

private void showShowcase() {
    ShowcaseView sv;

    ShowcaseView.ConfigOptions co = new ShowcaseView.ConfigOptions();
    co.hideOnClickOutside = false;
    sv = ShowcaseView.insertShowcaseView(gv.getChildAt(0), this.getSherlockActivity(), R.string.showcase_tile_title,
            R.string.showcase_tile, co);
    sv.animateGesture(400, 400, 20, 400);
    sv.animateGesture(20, 400, 400, 400);

    sv.setOnShowcaseEventListener(new OnShowcaseEventListener() {

        @Override
        public void onShowcaseViewShow(ShowcaseView showcaseView) {
            // TODO Auto-generated method stub
        }

        @Override
        public void onShowcaseViewHide(ShowcaseView showcaseView) {
            // TODO Auto-generated method stub
        }
    });
}
```

Basically, there's no error. ShowcaseView doesn't highlight anything, because what I am passing is null. My problem is that I don't know what to pass for it to showcase GridView at position 0.
